security/cert_manager_operator/cert-manager-operator-release-notes.adoc

```diff
@@ -12,6 +12,31 @@ These release notes track the development of {cert-manager-operator}.
 
 For more information, see xref:../../security/cert_manager_operator/index.adoc#cert-manager-operator-about[About the {cert-manager-operator}].
 
+[id="cert-manager-operator-release-notes-1-15-0"]
+== {cert-manager-operator} 1.15.0
+
+Issued: 2025-01-22
+
+Version `1.15.0` of the {cert-manager-operator} is based on the upstream cert-manager version `v1.14.5`. For more information, see the link:https://cert-manager.io/docs/releases/release-notes/release-notes-1.15/[cert-manager project release notes for v1.15].
+
+*Scheduling Overrides for {cert-manager-operator}*
+
+With this release, you can configure scheduling overrides for  {cert-manager-operator}, including the controller, webhook, and CA injector.
+
+For more information, see xref:../../security/cert_manager_operator/cert-manager-customizing-api-fields.adoc#cert-manager-scheduling-overrides[Configuring Scheduling Overrides for {cert-manager-operator}].
+
```

Comment:
Please add the following:
1. https://issues.redhat.com/browse/CM-436: operator no longer uses the redundant kube-rbac-proxy sidecar container.
2. https://issues.redhat.com/browse/OCPBUGS-23406: from 1.15.0 onwards the default and recommended operator OLM `installMode` is `AllNamespaces` (instead of the previously default `SingleNamespace`).